Dockerfiles/midstream/Dockerfile

```diff
@@ -1,20 +1,24 @@
 FROM registry.fedoraproject.org/fedora:33
-WORKDIR /project
-ARG OPERATOR_SDK_VERSION=v1.4.0
-ARG OPERATOR_TEST_PLAYBOOKS_TAG=v1.0.11
+ARG OPERATOR_SDK_VERSION=v1.9.0
 ARG UMOCI_VERSION=v0.4.5
-ADD ./fix_etc_passwd.sh /usr/bin/fix_etc_passwd.sh
+ENV ANSIBLE_CONFIG=/project/operator-test-playbooks/Dockerfiles/midstream/
+ENV ANSIBLE_LOCAL_TEMP=/tmp/
+ENV PATH="/operator-test-playbooks/Dockerfiles/midstream/:${PATH}"
+ADD extract-operator-bundle.yml validate-operator-bundle.yml /project/operator-test-playbooks/
```

Comment:
There seems to be an identical line in this same file on line 21

Reply:
Yes. Agreed